### ISSUE TITLE
e2e testing does not work.

### ISSUE THREAD

**Issue by KarelVendla** [nuxt/test-utils#720]
### Environment

------------------------------
- Operating System: Linux
- Node Version:     v18.18.0
- Nuxt Version:     3.9.3
- CLI Version:      3.10.0
- Nitro Version:    2.8.1
- Package Manager:  npm@10.2.3
- Builder:          -
- User Config:      devtools
- Runtime Modules:  -
- Build Modules:    -
------------------------------

### Reproduction

https://stackblitz.com/github/nuxt/test-utils/tree/main/examples/app-vitest

### Describe the bug

The default e2e app-vitest example does not work. It stalls on beforeAll hook.

`Error: Hook timed out in 120000ms.
If this is a long-running hook, pass a timeout value as the last argument or configure it globally with "hookTimeout".`

**Comment by danielroe** [comment]
Yes, browser testing doesn't work in stackblitz. The rest work fine.

![CleanShot 2024-01-18 at 16 42 15@2x](https://github.com/nuxt/test-utils/assets/28706372/952cee69-8cc2-47fb-8f58-a1af7ab74661)

**Comment by KarelVendla** [comment]
I ran the same exact reproduction and it took me forever?
![Capture](https://github.com/nuxt/test-utils/assets/10585924/161af18a-ae8b-4f23-9ac7-ed8d26e0417e)

**Comment by KarelVendla** [comment]
I also ran the same reproduction on. It went a lot faster. It seems the tests completed, but it still failed.
![Capture](https://github.com/nuxt/test-utils/assets/10585924/f0fb50c1-e73b-46c0-8b0d-92a097ea85c2)

**Comment by KarelVendla** [comment]
But the real problem in my case is that when for example I try to use the [browser e2e test](https://github.com/nuxt/test-utils/blob/main/examples/app-vitest/test/browser.e2e.spec.ts) in my local project it also stalls  and throws an error
`Error: Worker terminated due to reaching memory limit: JS heap out of memory`
or a hookTimeout

**Comment by danielroe** [comment]
Would you provide a reproduction? 🙏

**Comment by KarelVendla** [comment]
After debugging our local project that ran into this heap out of memory problem we found out that using [i18n](https://i18n.nuxtjs.org/) with 20 locales just doesn't cut it for e2e tests on a windows machine. The lazy loading doesn't seem to really work and all the locales are being loaded into the system on setup. We brought the locale count down to 2 for testing and we also turned off jit which shaved of about 20 seconds of the initial 100 second test time ..
But there is something else that is bothering it within our project that we haven't been able to figure out ..
So to conclude: We need to reconfigure and go over our project and find out the bottlenecks that are causing it.

But the thing that most surprises me is the fact that stackblitz results vary from computer to computer?? I am getting 164 second duration on stackblitz for the default test example.
